static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -212,16 +225,73 @@ function SelectProjectStep({
   );
 }
 
-function AndroidInstallSteps() {
+interface ProjectSdkUpdateProps {
+  minSdkVersion: string;
+  project: Project;
+  sdkUpdates: RequestState<ProjectSdkUpdates | null>;
+}
+function ProjectSdkUpdate(props: ProjectSdkUpdateProps) {
+  if (props.sdkUpdates.type !== 'resolved') {
+    return <div>{t('Verifying Sentry SDK')}</div>;
+  }
+
   return (
     <Fragment>
-      <li>
-        <StepTitle>{t('Update your projects SDK version')}</StepTitle>
-        <p>
+      <SDKUpdatesContainer>
+        <SdkUpdatesPlatformIcon platform={props.project.platform ?? 'unknown'} />
+        <span>{props.project.name}</span>
+      </SDKUpdatesContainer>
+      {props.sdkUpdates.data === null ? (
+        <SdkUpdatesText>
+          {t('This project is using the latest SDK version.')}
+        </SdkUpdatesText>
+      ) : (
+        <SdkUpdatesText>
           {t(
-            'Make sure your SDKs are upgraded to at least version 6.0.0 (sentry-android).'
+            'For profiling to function, we requires you to update your Sentry SDK to version %s or higher.',
```

Comment:
This check that `sdkUpdates.data` is not null might be too restrictive since they may be on a version of the SDK that supports profiling already and still see this message when there's a newer version. Although we likely want them to update to the latest, this message seems to imply that their current version doesn't support profiling.

Reply:
@Zylphrex Yes, that is true, but I dont know if we have any way to tell on which version the project is on exactly right now (or I missed it)

Reply:
unsure if we already have a way of parsing SDK version on the frontend but it looks like the SDK updates endpoint returns the current SDK version.